yt/data_objects/derived_quantities.py

```diff
@@ -588,23 +591,25 @@ class Extrema(DerivedQuantity):
 
     """
 
-    def count_values(self, fields, non_zero):
+    def count_values(self, fields, non_zero, check_finite):
```

Comment:
given the wild diversity that already exist in the various implementations of `DerivedQuantity.count_values` and how common boolean positionally-allowed arguments are, let's dodge this time :)
```suggestion
    def count_values(self, fields, non_zero, *, check_finite):
```

Reply:
ya! agreed. changing just this line causes an error because down below in ``__call__`` I currently have it passing along `check_finite` as an arg rather than kwarg: 

```python
rv = super().__call__(fields, non_zero, check_finite)
```
So I'll push up a change with your suggestion and the other line fixed as well (after my meeting that starts in a minute....)